src/electionguard-ui/ElectionGuard.UI/ViewModels/CreateMultiTallyViewModel.cs

```diff
@@ -0,0 +1,353 @@
+        ElectionGuardException.ThrowIfNull(keyCeremony, $"Could not load key ceremony {election.Election.KeyCeremonyId}");
+
+        var tallyName = $"{election.Election.Name} {AppResources.TallyText}";
+
+        TallyRecord newTally = new()
+        {
+            Name = tallyName,
+            ElectionId = election.Election.ElectionId,
+            KeyCeremonyId = election.Election.KeyCeremonyId,
+            Quorum = keyCeremony.Quorum,
+            NumberOfGuardians = keyCeremony.NumberOfGuardians,
+            CastBallotCount = ballotCount,
+            ChallengedBallotCount = challengedCount,
+            SpoiledBallotCount = spoiledCount,
+            State = TallyState.PendingGuardiansJoin,
+            MultiTally = multi,
+        };
+
+        _ = await _tallyService.SaveAsync(newTally);
+
+        return newTally.TallyId;
+    }
+
+
+    [RelayCommand(CanExecute = nameof(TalliesSelected))]
+    private async Task CreateTallies()
+    {
+        if(SelectedElections.Count == 1)
+        {
+            // do a normal tally for a single election
+            var election = SelectedElections.First() as ElectionItem;
+
+            var tallyId = await CreateNewTally(election);
+
+            // go to the processing page
+            await NavigationService.GoToPage(typeof(TallyProcessViewModel), new Dictionary<string, object>
+            {
+                { TallyProcessViewModel.CurrentTallyIdParam, tallyId }
+            });
+        }
+        else
+        {
+            List<string> tallys = new();
+            // create tally records for each election selected
+            foreach (var item in SelectedElections)
+            {
+                var election = item as ElectionItem;
+                var tallyId = await CreateNewTally(election, true);
+                tallys.Add(tallyId);
+            }
+
+            // create a multi-tally record for the db
+            MultiTallyRecord multiTallyRecord = new()
+            {
+                Name = $"MultiTally for {SelectedKeyCeremony.Name}",
+                KeyCeremonyId = SelectedKeyCeremony.KeyCeremonyId,
+                TallyIds = tallys,
+                ResultsPath = CurrentResultsPath
+            };
+
+            _ = await _multiTallyService.SaveAsync(multiTallyRecord);
+
+            // go to the processing page
+            await NavigationService.GoToPage(typeof(TallyProcessViewModel), new Dictionary<string, object>
+            {
+                { TallyProcessViewModel.MultiTallyIdParam, multiTallyRecord.MultiTallyId }
+            });
+        }
+    }
+
+    private async Task JoinTally(string tallyId, bool join = true)
+    {
+        // join the tally
+        var joiner = new TallyJoinedRecord()
+        {
+            TallyId = tallyId,
+            GuardianId = UserName!, // can assume not null, since you need to be signed into 
+            Joined = join,
+        };
+
+        await _tallyJoinedService.JoinTallyAsync(joiner);
+    }

```

Comment:
Future: Definitely have seen this before... there's definitely a case for having a layer of objects distinct from the DAO records we use, where we would stick this data.